src/pages/item/item.html

```diff
@@ -31,6 +31,6 @@
       </ion-item>
 		</ion-list>
 
-		<button ion-button (click)="onSave(itemForm)" type="submit" block>Save</button>
+		<button ion-button (click)="onSave(itemForm)" type="submit" block [disabled]="!selectedBrand || !selectedModel || !selectedCategory">Save</button>
```

Comment:
The conditional for `disabled` is too simplistic, isn't it? What if any of the fields contain just whitespace, for example?

You may be able to implement part of this functionality with simple HTML5 form validation and the `required` attribute.